test/reducers/dataReducer.test.js

```diff
@@ -84,3 +85,11 @@ test('data reducer EVENTS_LOAD_FAILED action', t => {
   t.equal(actual.error.message, 'Error: Something failed', 'sets error message by serialising the error')
   t.end()
 })
+
+test('data reducer HIDE_ERROR action', t => {
+  const expected = {errorVisible: false}
+  const actual = reducer({errorVisible: true},
```

Comment:
Could you pass more complete state to the reducer to verify rest of the state won't be affected? All this test says is that I could return just `{errorVisible: true}` from the reducer and be fine with it.